<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkNetworkPrefWidget</class>
 <widget class="QWidget" name="WkNetworkPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="toolTip">
        <string>Multicast address</string>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="UtQtIpv4LineEdit" name="addressEdit">
       <property name="toolTip">
        <string>Multicast address</string>
       </property>
       <property name="text">
        <string>255.255.255.255</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="interfaceLabel">
       <property name="toolTip">
        <string>Local socket</string>
       </property>
       <property name="text">
        <string>Interface:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="interfaceList">
         <property name="toolTip">
          <string>Local socket</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addInterface">
         <property name="toolTip">
          <string>Add Interface</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="portSpinBox">
       <property name="minimum">
        <number>1025</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Refresh the interface list by re-opening preferences.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtIpv4LineEdit</class>
   <extends>QLineEdit</extends>
   <header>UtQtNetwork.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
